nBTC/sources/nbtc.move

```diff
@@ -120,7 +127,8 @@ fun init(witness: NBTC, ctx: &mut TxContext) {
         tx_ids: table::new<vector<u8>, bool>(ctx),
         bitcoin_lc: @bitcoin_lc.to_id(),
         fallback_addr: @fallback_addr,
-        nbtc_bitcoin_pkh,
+        nbtc_bitcoin_pkhs: vec_set::singleton(nbtc_bitcoin_pkh),
+        nbtc_active_keys: vec_set::singleton(nbtc_bitcoin_pkh),
```

Comment:
we need to track balance per key. 
- So add a mapping (using vector map - we don't need table for that).
- we don't need all/ not active keys. we only need active ones. And let's call them `bitcoin_pkhs`.
- when we delete a key, we need to check if balance is zero. Then we delete it.

On top of that:
- Add TODOs
- Add TODO to expand the `bitcoin_pkhs` type to support more types.
- Add TODO to rebalance the wallet (move funds from one active key to anther on the Bitcoin side).